apps/server/src/modules/files/job/sync-files.console.ts

```diff
@@ -17,9 +17,13 @@ export class SyncFilesConsole {
 		private logger: Logger
 	) {}
 
-	@Command({ command: 'tasks [aggregationSize] [numParallelPromises]' })
-	async syncFilesForTasks(aggregationSize = 5000, numParallelPromises = 50) {
-		await this.syncFilesUc.syncFilesForParentType(FileRecordParentType.Task, aggregationSize, numParallelPromises);
+	@Command({ command: 'submissions [aggregationSize] [numParallelPromises]' })
```

Comment:
I was surprised that you hardcoded it. I thought you would do it this way. Did you think about that, too? 

```suggestion
	@Command({ command: 'appended [type] [aggregationSize] [numParallelPromises]' })
```

Reply:
We decided that level of generalization is enough for this code.